cmd/umoci/utils.go

```diff
@@ -245,3 +246,37 @@ func Stat(ctx context.Context, engine casext.Engine, manifestDescriptor ispec.De
 
 	return stat, nil
 }
+
+// writeMtree writes an mtree of the rootfs in the given bundle path into the
+// file by the given name, using the supplied fsEval method
+func writeMtree(mtreeName string, bundlePath string, fsEval mtree.FsEval) error {
```

Comment:
I'd prefer this be named `generateBundleManifest` or something like that? Also if we're going to do the `: -> _` rename we might as well do it inside this function.